lineman/spec-e2e/proposal_spec.coffee

```diff
@@ -1,87 +1,62 @@
 describe 'Proposals', ->
 
   DiscussionPage = require './helpers/discussion_page.coffee'
+  proposalsHelper = require './helpers/proposals_helper.coffee'
   page = new DiscussionPage
```

Comment:
would be good if DiscussionPage had a 'new' at the beginning of it.. so we end up with discussionPage = require ... and no need for new DiscissionPage here